tests/integrations/test_crossed_integrations.py

```diff
@@ -87,6 +86,7 @@ def test_produce(self):
 
     @missing_feature(library="python")
     @missing_feature(library="java")
+    @missing_feature(library="nodejs")
```

Comment:
It would be good to add the reason.
Even the version (if possible)

Reply:
the reason is that python does not propagate context in a normal way, so no one else can hook up with it :/

Reply:
Added the "reason" field